schemata/applicant_details.json

```diff
@@ -0,0 +1,49 @@
+{
+  "$schema": "http://json-schema.org/draft-04/hyper-schema",
+  "id": "/schemata/applicant_details",
+  "title": "Applicant Details",
+  "stability": "prototype",
+  "description": "The applicant details stores information about the applicant, only some of those information are editable.",
+  "links": [
+    {
+      "description": "Update a batch of Applicant Details, this endpoint is **limited to 1000 records per call**.",
```

Comment:
We should probably say **limited to 1000 Applicants per call** ?
Details are 1:1 and I think it is a little clearer. what do you think?

Reply:
👍